cypress/e2e/brewery/ScenarioParameters-additional-advanced.cy.js

```diff
@@ -105,6 +116,7 @@ describe('Additional advanced scenario parameters tests', () => {
     BreweryParameters.switchToAdditionalParametersTab();
     BreweryParameters.getCommentInput().should('value', INIT_VALUES.comment);
     BreweryParameters.getAdditionalDateInput().should('value', INIT_VALUES.additionalDate);
+    BreweryParameters.getScenarioToCompareInput().should('have.text', '');
```

Comment:
I don't see this function in `BreweryParameters` export (the same on line 119) :thinking: